release: sourcegraph@3.35.0 (#29260)
sourcegraph/sourcegraph: doc/admin/updates/docker_compose.md

<!-- GENERATE UPGRADE GUIDE ON RELEASE (release tooling uses this to add entries) -->

## 3.34 -> 3.35

> The `query-runner` service has been decommissioned in the 3.35.0 release and will be removed during the upgrade.

Author: I'm a big jerk and didn't confirm what I was saying.

The docker-compose command needs to be run with `--remove-orphans` in order to delete query-runner.

Author: We do mention that in the upgrade docs but it's good to call out. Also, probably need to preserve the `follow the [standard upgrade procedure](../install/docker-compose/operations.md#upgrade) to upgrade your deployment` bit here.

> The `query-runner` service has been decomissioned in the 3.35.0 release. You can safely remove the `query-runner` service from your installation.

Author: docker-compose installations don't need to do anything special so maybe something like:
```suggestion
The `query-runner` service has been decommissioned in the 3.35.0 release and will be removed during the upgrade.
```
